CONTRIBUTING.md

```diff
@@ -5,6 +5,74 @@ Welcome, and thank you for your interest in contributing to iTwin.js!
 There are many ways to contribute.
 The goal of this document is to provide a high-level overview of how you can get involved.
 
+- [Contributing to iTwin.js](#contributing-to-itwinjs)
+  - [Repo Setup](#repo-setup)
+  - [Source Code Edit Workflow](#source-code-edit-workflow)
+    - [Other NPM Scripts](#other-npm-scripts)
+  - [Asking Questions](#asking-questions)
+  - [Providing Feedback](#providing-feedback)
+  - [Reporting Issues](#reporting-issues)
+    - [Look For an Existing Issue](#look-for-an-existing-issue)
+    - [Writing Good Bug Reports and Feature Requests](#writing-good-bug-reports-and-feature-requests)
+    - [Follow Your Issue](#follow-your-issue)
+  - [Contributing Guidelines](#contributing-guidelines)
+    - [Contributor License Agreement (CLA)](#contributor-license-agreement-cla)
+    - [Pull Requests](#pull-requests)
+    - [Types of Contributions](#types-of-contributions)
+  - [Frequently Asked Questions](#frequently-asked-questions)
+    - [Rush commands take too long, is there a way to speed up my contribution workflow?](#rush-commands-take-too-long-is-there-a-way-to-speed-up-my-contribution-workflow)
+    - [Do I have to rebuild all packages in the repo, even those I didn't work on?](#do-i-have-to-rebuild-all-packages-in-the-repo-even-those-i-didnt-work-on)
+    - [A subdirectory can not find a node\_modules file or directory](#a-subdirectory-can-not-find-a-node_modules-file-or-directory)
+    - [Updating dependencies/devDependencies on packages within the monorepo](#updating-dependenciesdevdependencies-on-packages-within-the-monorepo)
+    - [Updating dependencies/devDependencies on packages external to monorepo](#updating-dependenciesdevdependencies-on-packages-external-to-monorepo)
+
+## Repo Setup
+
+This repository is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) that holds the source code to multiple iTwin.js npm packages. It is built using [Rush](http://rushjs.io/).
+
+Each package has its own **node_modules** directory that contains symbolic links to *common* dependencies managed by Rush.
+
+1. Install dependencies: `rush install`
+2. Clean: `rush clean`
```

Comment:
honestly clean is nice to use when there are issues, but not always needed

Reply:
yeah, removed